README.md

```diff
@@ -156,3 +156,8 @@ if (thisRepo.isAwesome()) {
 ---
 
 [![built with love](https://forthebadge.com/images/badges/built-with-love.svg)](https://github.com/vinitshahdeo/) [![powered by water](https://forthebadge.com/images/badges/powered-by-water.svg)](https://github.com/vinitshahdeo/) [![smile please](https://forthebadge.com/images/badges/makes-people-smile.svg)](https://github.com/vinitshahdeo/)
+
+<img src="https://img.shields.io/badge/Made%20with-Node.js-orange?style=for-the-badge" alt="Node.js">
```

Comment:
@AdityaAshvin Avoid `<html>` tags, please use `.md` syntax.

Reply:
Made the changes. @vinitshahdeo